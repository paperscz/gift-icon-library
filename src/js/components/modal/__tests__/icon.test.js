import React from 'react';
import { mount } from 'enzyme';

import Icon from '../icon';

describe('The icon component', () => {
  let component;

  it('should render with no props', () => {
    component = mount(<Icon />);
    expect(component).toMatchSnapshot();
  });

  it('should render with props', () => {
    component = mount(<Icon name="test name" svg="/path/to/svg" />);
    expect(component).toMatchSnapshot();
  });

});
