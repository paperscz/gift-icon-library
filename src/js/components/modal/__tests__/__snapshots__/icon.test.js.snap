// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The icon component should render with no props 1`] = `
<Icon>
  <div
    className="icon"
  >
    <h4 />
    <img />
    <pre />
  </div>
</Icon>
`;

exports[`The icon component should render with props 1`] = `
<Icon
  name="test name"
  svg="/path/to/svg"
>
  <div
    className="icon"
  >
    <h4>
      test name
    </h4>
    <img
      alt="test name"
      src="/path/to/svg"
    />
    <pre>
      /path/to/svg
    </pre>
  </div>
</Icon>
`;
